app_dart/lib/src/request_handlers/flaky_handler_utils.dart

```diff
@@ -425,10 +454,56 @@ String _issueBuildLink({String builder, String build}) {
   return Uri.encodeFull('$_buildPrefix$builder/$build');
 }
 
+Team _teamFromString(String teamString) {
+  switch (teamString) {
+    case 'flutter/framework':
+      return Team.framework;
+    case 'flutter/engine':
+      return Team.engine;
+    case 'flutter/tool':
+      return Team.tool;
+    case 'flutter/web':
+      return Team.web;
+  }
+  return Team.unknown;
+}
+
+String _getTeamLabelFromTeam(Team team) {
+  switch (team) {
+    case Team.framework:
+      return kFrameworkLabel;
+    case Team.engine:
+      return kEngineLabel;
+    case Team.tool:
+      return kToolLabel;
+    case Team.web:
+      return kWebLabel;
+    case Team.unknown:
+      return null;
+  }
```

Comment:
A return is needed out of the switch block.

Reply:
fixed